src/schedule.rs

```diff
@@ -649,4 +713,49 @@ mod test {
         assert!(schedule_1.timeunitspec_eq(&schedule_2));
         assert!(schedule_3.timeunitspec_eq(&schedule_4));
     }
+
+    #[test]
+    fn test_dst_ambiguous_time_after() {
+        use chrono_tz::Tz;
+
+        let schedule_tz: Tz = "America/Chicago".parse().unwrap();
+        let dt = schedule_tz
+            .ymd(2022, 11, 5)
+            .and_hms_opt(23, 30, 0)
+            .unwrap();
+        let schedule = Schedule::from_str("0 0 * * * * *").unwrap();
+        let times = schedule.after(&dt).map(|x| x.to_string()).take(5).collect::<Vec<_>>();
+        let expected_times = [
+            "2022-11-06 00:00:00 CDT".to_string(),
+            "2022-11-06 01:00:00 CDT".to_string(),
+            "2022-11-06 01:00:00 CST".to_string(), // 1 AM happens again
+            "2022-11-06 02:00:00 CST".to_string(),
+            "2022-11-06 03:00:00 CST".to_string(),
+        ];
+
+        assert_eq!(times.as_slice(), expected_times.as_slice());
+    }
+
+    #[test]
+    fn test_dst_ambiguous_time_before() {
+        use chrono::NaiveDateTime;
+        use chrono_tz::Tz;
+
+        let schedule_tz: Tz = "America/Chicago".parse().unwrap();
+        let dt = schedule_tz
+            .ymd(2022, 11, 6)
+            .and_hms_opt(3, 30, 0)
+            .unwrap();
+        let schedule = Schedule::from_str("0 0 * * * * *").unwrap();
+        let times = schedule.after(&dt).map(|x| x.to_string()).rev().take(5).collect::<Vec<_>>();
+        let expected_times = [
+            "2022-11-06 03:00:00 CST".to_string(),
+            "2022-11-06 02:00:00 CST".to_string(),
+            "2022-11-06 01:00:00 CST".to_string(),
+            "2022-11-06 01:00:00 CDT".to_string(), // 1 AM happens again
+            "2022-11-06 00:00:00 CDT".to_string(),
+        ];
+
+        assert_eq!(times.as_slice(), expected_times.as_slice());
+    }
```

Comment:
These tests verify that we're not missing an hour at the daylight savings time change for both forward and backward iteration.